uint32 timestamp
uint8 n_frequencies

# List of frequencies, in Hz
float32[] frequencies

# Spatial spectrum data for different frequencies. 
float32[] spectrum_vect
